index.js

```diff
@@ -1,5 +1,6 @@
 const {transcodeAudioFile} = require('./lib/audio');
```

Comment:
Can we change the code to reflect simply "media" instead of audio or video. For example 'lib/media', transcodeMedia etc.